res/css/_spacing.scss

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// 1rem :: 10px
```

Comment:
should these be relative? In practice our font size changes depending on a wide variety of factors, so concerned that it will cause overly-padded UI instead of the intended effect.

Reply:
I was thinking about the case of zooming the browser, but I see now that we scale up the body font size based on the font size in settings. Will switch to px.